### DISCUSSION TITLE
python-httpx

### DISCUSSION THREAD

**Discussion by cbix** [projectdiscovery/httpx#693] [General] [answered]
Someone on IRC raised the concern that besides this amazing project, there is [python-httpx](https://github.com/encode/httpx), which also installs an `httpx` executable. This may be a problem for packagers of Linux distros because they would have to either make the two (unrelated) packages conflict or give one of them a different name.

A similar situation happened in Ubuntu a while ago where in Focal the `docker` package referred to a project that later was renamed to `wmdocker` to avoid confusion with Docker 😅

Have you considered renaming the project or the installed binary to solve this conflict? Maybe something like `httpx-cli` or `go-httpx`?

**Comment by forgedhallpass** [comment, marked as answer]
Thank you for pointing it out, we know about the situation.

Indeed, there are multiple people having the same issue:
* https://github.com/projectdiscovery/httpx/discussions/699
* https://github.com/projectdiscovery/httpx/discussions/774
* https://github.com/projectdiscovery/httpx/discussions/775

That being said, we have multiple tools using the same naming convention (e.g. [`dnsx`](https://github.com/projectdiscovery/dnsx), [`tlsx`](https://github.com/projectdiscovery/dnsx)), hence at this point we are not eager to change it.

For the moment, people can use the above mentioned fixes/workarounds.

**Reply by forgedhallpass** [reply]
Update: in Kali Linux, our version of `httpx` is available under the [`httpx-toolkit`](https://www.kali.org/tools/httpx-toolkit/) name.
